users/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+from categories.models import Category
+
+class User(models.Model): 
+    name       = models.CharField(max_length=45)
+    email      = models.EmailField(unique=True)
+    kako_id    = models.CharField(max_length=45)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta: 
+        db_table = 'uesrs'
+
+class Host(models.Model): 
+    phone_number = models.CharField(max_length=45, unique=True)
+    carrer       = models.IntegerField()
+    price        = models. IntegerField()
```

Comment:
이런 부분 있으면 migration 이 정상적으로 되지 않았을텐데 하고 찾아보니 migration 파일들이 하나도 없네요. 
모델 생성, 수정사항이 생겼을경우 꼭 migration 파일이 함께 올라와야합니다.
```suggestion
    price        = models.IntegerField()
```

Reply:
컨펌 받고 마이그레이션 하려고 했습니다! 마이그레이트하고 pr 올려야하나요?